stable/opsportal/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kommander-ui
-    version: "6.81.7"
+    version: "6.84.1"
```

Comment:
this is reverting?

Reply:
no, upgrading 81 -> 84? unless I messed this up, I've looked at too many version numbers today

Reply:
lgtm